package.json

```diff
@@ -33,17 +33,18 @@
     "lint": "eslint src"
   },
   "dependencies": {
+    "@babel/helper-module-imports": "7.0.0-beta.31",
```

Comment:
Are you locking the version in for all the babel deps on purpose? It's probably a better idea to add a leading `^` so we get any future updates.

Reply:
Because this is a semver prerelease version, Babel isn't currently guaranteeing compatibility between beta releases. We're certainly not going out of our way to break things, but it's always possible.